src/Codeception/Module/Yii2.php

```diff
@@ -55,6 +55,8 @@ public function _before(\Codeception\TestCase $test)
     {
         $this->client = new \Codeception\Lib\Connector\Yii2();
         $this->client->configFile = \Codeception\Configuration::projectDir().$this->config['configFile'];
+        $this->client->setServerParameter('HTTP_HOST', parse_url(\Codeception\Configuration::config()['config']['test_entry_url'], PHP_URL_HOST));
```

Comment:
Oh, this is not an official specification, so I'd ask you to wrap your code into `if` block, where you check this parameter (as well as `config` section) actually exists